.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python: "2.7"
+before_install:
```

Comment:
you can get this travis config onto docker if you go with 
`sudo: false` and specify a node version.

See [this PR](https://github.com/edx/edx-analytics-dashboard/pull/417) for an example, and note that you'll probably want a more recent version of node.